### PR TITLE
Node Editor: QoL Fixes

### PR DESCRIPTION
**Pull request by blessedcoolant**
- Make the viewport fit to view on Init
- Update Reload Schema to an icon.

### CODE REVIEW

File: invokeai/frontend/web/src/features/nodes/components/Flow.tsx [hunk 1]
```diff
@@ -67,6 +69,10 @@ export const Flow = () => {
     dispatch(connectionEnded());
   }, [dispatch]);
 
+  const onInit: OnInit = useCallback((v: ReactFlowInstance) => {
+    if (v) v.fitView();
```
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/nodes/components/Flow.tsx:73]: Pls use full conditional syntax eg `if (...) {...}` - it's way too easy for bugs to creep in without the curly braces

File: invokeai/frontend/web/src/features/nodes/components/ui/ReloadSchemaButton.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+
+  const handleReloadSchema = useCallback(() => {
+    dispatch(receivedOpenAPISchema());
+  }, [dispatch]);
+
+  return (
+    <IAIIconButton
+      icon={<BiRefresh />}
```
Comment (psychedelicious) [inline invokeai/frontend/web/src/features/nodes/components/ui/ReloadSchemaButton.tsx:18]: Can we use the corresponding FontAwesome icon here? Trying to use FontAwesome everywhere so we have both consistency in style and in sizing - different icon packs have their own unique size, padding and style, that makes using different packs look wonky.

**Review by psychedelicious — CHANGES_REQUESTED** [review]

File: invokeai/frontend/web/src/features/nodes/components/Flow.tsx (hunk 1 above)
Comment (blessedcoolant) [reply invokeai/frontend/web/src/features/nodes/components/Flow.tsx:73]: Curly brace lover I see. Fixed.

Comment (psychedelicious) [reply invokeai/frontend/web/src/features/nodes/components/Flow.tsx]: Yes :P I've borked myself too many times with the more compact syntax!

**Review by psychedelicious — APPROVED** [review]

**Pull request merged**